service/project.go

```diff
@@ -581,6 +582,12 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		containerSizes[key] = containerResource
 	}
 
+	containerCredentials := map[string]model.ContainerCredential{}
+	for key, apiContainerCredential := range responseRef.ContainerCredentials {
+		containerCredential := apiContainerCredential.ToService()
```

Comment:
This loop is just meant to construct a new `map[string]model.ContainerCredential` from the `map[string]restModel.APIContainerCredential` type sent in from the request so that it can be saved to the ProjectRef, not meant to perform any validation.